modules/osm/node.js

```diff
@@ -120,7 +120,13 @@ const prototype = {
 
             var nodeIds = {};
             resolver.parentWays(this)
-                .filter(way => osmShouldRenderDirection(this.tags, way.tags))
+                .filter(way => {
+                    // check for outbound one-way ways from stop signs
+                    const isOutbound = way.tags.oneway === 'yes' && way.nodes[0] === this.id;
+                    const isStopSign = this.tags.highway === 'stop';
```

Comment:
```suggestion
                    const isStopSign = this.tags.highway === 'stop' && this.tags.stop === 'all';
```
I think we need to check for `stop=all`, because otherwise this codes breaks the logic for nodes that are explicitly tagged as `direction=backward`.

This is also a good reminder of https://github.com/openstreetmap/id-tagging-schema/pull/1539 which should probably be extended to other countries that don't have `stop=all`